src/hooks/usePermit/rpc.ts

```diff
@@ -0,0 +1,24 @@
+import type { Web3Provider } from '@ethersproject/providers'
+import type { TRSVResponse } from './types'
+import { splitSignatureToRSV } from './utils'
```

Comment:
Disadvantage of having "everything in Utils" is that now you have cycle deps between rpc and utils.
"call" imported in utils
This one imported in rpc

Reply:
I will fixed this circular dependency in next release, thanks 🙏